packages/devtools-local-toolbox/config/local.json

```diff
@@ -0,0 +1,20 @@
+{
+  "hotReloading": true,
+  "chrome": {
+    "debug": true
+  },
+  "logging": {
+    "firefoxProxy": false,
+    "actions": false
+  },
+  "redux-devtools": {
+    "enabled": false
+  },
+  "firefox": {
+    "geckoDir": "/Users/jlaster/src/mozilla/gecko"
+  },
+  "features": {
+    "prettyPrint": true,
+    "watchExpressions": true
```

Comment:
A lot of these config values are specific to the debugger. Well, I suppose this highlights the difference between features flags and config values. The former are debugger-specific, while the latter aren't. I think we should split them up somehow.